docs/run-operator-locally.md

```diff
@@ -12,6 +12,7 @@ Being able to run and build the binary locally can help with faster feedback-cyc
     - `KUBECONFIG` environment variable pointing at a file
     - **Note**: either of these are necessary to be able to run the operator locally
 - Have a folder `.community-operator-dev`
+- *Optional - if you want to export the environment variables, you can run the following command*: `source .community-operator-dev/local-test.export.env`
```

Comment:
Can we mention here that we are creating those exports with the make target: `make generate-env-file`?